FabNeso.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+#
+# This source file is part of the FabSim software toolkit, which is
+# distributed under the BSD 3-Clause license.
+# Please refer to LICENSE for detailed information regarding the licensing.
+#
+# This file contains FabSim definitions specific to FabNeso
+#
+# authors: Duncan Leggat
+
```

Comment:
```suggestion
from pathlib import Path

```

For robust path handling.